entrypoint.sh

```diff
@@ -9,8 +9,14 @@ get_from_event() {
   jq -r "$1" "${GITHUB_EVENT_PATH}"
 }
 
+# Test required inputs are set
+if [ -z "$INPUT_RUN_ID" ]; then
+  echo "Missing input run_id"
+fi
```

Comment:
@alexjurkiewicz shouldn't we fail the script here, since the run id variable is marked as required?

Reply:
You are right. I'll fix this.